moviedb/index.d.ts

```diff
@@ -1,24 +1,44 @@
 // Type definitions for MovieDB
 // Project: https://github.com/danzajdband/moviedb
 // Definitions by: Basarat Ali Syed <https://github.com/basarat>
+// Definitions by: Pavel Puchkov <https://github.com/0x6368656174>
```

Comment:
Add this on the same line: `// Definitions by: Basarat Ali Syed <https://github.com/basarat>, Pavel Puchkov <https://github.com/0x6368656174>`